src/test/ui/consts/const-for.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0015]: calls in constants are limited to constant functions, tuple structs and tuple variants
+  --> $DIR/const-for.rs:5:14
+   |
+LL |     for _ in 0..5 {}
+   |              ^^^^
```

Comment:
I guess we should start printing the function's name in the diagnostic. Very uninformative this error XD